rtl/eclrtl/eclrtl.cpp

```diff
@@ -2371,6 +2486,55 @@ ECLRTL_API void rtlTrimUtf8All(unsigned &tlen, char * &tgt, unsigned slen, const
     rtlUnicodeToUtf8X(tlen, tgt, trimLen, trimText.getustr());
 }
 
+void rtlTrimUtf8WS(unsigned & tlen, char * & tgt, unsigned slen, const char * src, bool left, bool all, bool right)
+{
+    if (all)
+    {
+        //Go via unicode because of possibility of combining accents etc.
+        rtlDataAttr temp1(slen*sizeof(UChar));
```

Comment:
horribly, this could have problems if the utf8 contains characters which map to surrogate pairs in utf8.  Safer to use rtlUtf8ToUnicodeX.  It would be good to have an example that did include these weird characters....